modules/eco-node-health-check-operator-about.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/node-health-check-operator-installation.adoc
+
+[id="about-node-health-check-operator_{context}"]
+= About the Node Health Check Operator
+
+The Node Health Check Operator deploys the `NodeHealthCheck` controller, which in turn creates the `NodeHealthCheck` custom resource (CR). The Node Health Check Operator also installs the Poison Pill Operator as a default remediation provider.
+
+The Operator uses the controller to detect the health of a node in the cluster. When the controller detects an unhealthy node, it creates a `NodeHealthCheck` custom resource (CR), which defines a set of criteria and thresholds to determine the node's health.
+
+The `NodeHealthCheck` CR  creates the `PoisonPillRemediation` CR, which triggers the Poison Pill Operator to remediate the unhealthy node. 
+
+The `NodeHealthCheck` CR resembles the following YAML file:
+
+[source,yaml]
+----
+apiVersion: remediation.medik8s.io/v1alpha1
+kind: NodeHealthCheck
+metadata:
+  name: nodehealthcheck-sample
+  namespace: openshift-operators
+spec:
+  maxUnhealthy: 49% <1>
+  remediationTemplate: <2>
+    apiVersion: poison-pill.medik8s.io/v1alpha1
+    name: group-x
+    namespace: openshift-operators
+    kind: PoisonPillRemediationTemplate
+  selector: <3>
+    matchExpressions:
+      - key: node-role.kubernetes.io/worker
+        operator: Exists
+  unhealthyConditions: <4>
+    - type: Ready
+      status: "False"
+      duration: 300s <5>
+    - type: Ready
+      status: Unknown
+      duration: 300s <5>
+----
+
+<1> Specifies the amount (in percentage) of nodes allowed to be concurrently remediated in the targeted pool. If the number of unhealthy nodes exceeds the limit set by `maxUnhealthy`, remediation is not performed. The default value is 49%.
+<2> Specifies a remediation template from the remediation provider. For example, from the Poison Pill Operator. 
+<3> Specifies a `selector` that matches labels or expressions that you want to check. The default value is empty, which selects all nodes.
+<4> Specifies a list of the conditions that determine whether a node is considered unhealthy. 	
+<5> Specifies the timeout duration for a node condition. If a condition is met for the duration of the timeout, the node will be remediated. Long timeouts can result in long periods of downtime for a workload on an unhealthy node.
+
+[id="understanding-nhc-operator-workflow_{context}"]
+== Understanding the Node Health Check Operator workflow
+
+When a node is identified as unhealthy, the Operator checks how many other nodes are unhealthy. If the number of unhealthy nodes is less than the amount that is specified in the `maxUnhealthy` field of the `NodeHealthCheck` CR, the controller creates an external remediation template depending on its remediation provider. The controller then updates the node's health status.
```

Comment:
its not creating a template but rather using the template details to create a CR out of it.